src/components/new-request/business-lookup.vue

```diff
@@ -5,13 +5,13 @@
     id="business-lookup"
   >
     <v-autocomplete
+      v-model:search-input="searchField"
```

Comment:
`lint` change.

Reply:
The new code looks like the Vue3-compatible change. Does it still work in Vue2?

Reply:
Thank you so much for catching this Sev. You are absolutely right. It doesn't work. I reverted this change.